### PR TITLE
squeezebox add sensors

### PR DESCRIPTION
**Pull request by pssc**
## Proposed change

Add docs for new squeezebox senor platform

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/125349
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Enhanced Squeezebox integration with support for sensor entities.
	- Added a new "Sensors" section detailing various metrics for monitoring media player performance, including last scan date, player counts, total albums, total artists, total duration, total genres, and total songs.

These updates provide users with improved insights into their media service's status and enhance overall functionality.

### CODE REVIEW

File: source/_integrations/squeezebox.markdown [hunk 1]
```diff
@@ -56,6 +57,17 @@ transporter_toslink:
...
+### Sensors
+
+- **Last scan**: Date of the last library scan.
+- **Player count**: Players on the service.
+- **Player count off service**: Players not on this service.
+- **Total albums**: Albums currently in the service.
+- **Total artists**: Artists currently in the service.
+- **Total duration**: Duration of all tracks in service.
+- **Total genres**: Genres used in current service.
+- **Total songs**: Number of msuic files currently in service.
```
Comment (c0ffeeca7) [inline source/_integrations/squeezebox.markdown]:
```suggestion
- **Total songs**: Number of music files currently in service.
```

File: source/_integrations/squeezebox.markdown [hunk 2]
```diff
@@ -56,6 +57,17 @@ transporter_toslink:
...
 - **Library rescan**:  The music library is currently being scanned by LMS (depending on the type of scan, some content may be unavailable).
 
+### Sensors
+
+- **Last scan**: Date of the last library scan.
+- **Player count**: Players on the service.
+- **Player count off service**: Players not on this service.
+- **Total albums**: Albums currently in the service.
+- **Total artists**: Artists currently in the service.
+- **Total duration**: Duration of all tracks in service.
```
Comment (c0ffeeca7) [inline source/_integrations/squeezebox.markdown]: What is the time format used here? Minutes and seconds, MM:SS? Hours, minutes, and seconds?

File: source/_integrations/squeezebox.markdown [hunk 3]
```diff
@@ -56,6 +57,17 @@ transporter_toslink:
 - **Needs restart**:  Server Service needs to be restarted (typically, this is needed to apply updates).
 - **Library rescan**:  The music library is currently being scanned by LMS (depending on the type of scan, some content may be unavailable).
 
+### Sensors
+
+- **Last scan**: Date of the last library scan.
+- **Player count**: Players on the service.
+- **Player count off service**: Players not on this service.
```
Comment (c0ffeeca7) [inline source/_integrations/squeezebox.markdown]:
```suggestion
- **Player count**: Number of players on the service.
- **Player count off service**: Number of players not on this service.
```

File: source/_integrations/squeezebox.markdown [hunk 4]
```diff
@@ -56,6 +57,17 @@ transporter_toslink:
...
+- **Total albums**: Albums currently in the service.
```
Comment (c0ffeeca7) [inline source/_integrations/squeezebox.markdown]:
```suggestion
- **Total albums**: Total number of albums currently available in the service.
```

File: source/_integrations/squeezebox.markdown [hunk 5]
```diff
@@ -56,6 +57,17 @@ transporter_toslink:
...
+- **Total artists**: Artists currently in the service.
```
Comment (c0ffeeca7) [inline source/_integrations/squeezebox.markdown]:
```suggestion
- **Total artists**: Total number of artists currently available in the service.
```

File: source/_integrations/squeezebox.markdown [hunk 6]
```diff
@@ -56,6 +57,17 @@ transporter_toslink:
...
+- **Total duration**: Duration of all tracks in service.
+- **Total genres**: Genres used in current service.
```
Comment (c0ffeeca7) [inline source/_integrations/squeezebox.markdown]:
```suggestion
- **Total genres**: Total number of genres used in current service.
```

File: source/_integrations/squeezebox.markdown [hunk 7]
```diff
@@ -56,6 +57,17 @@ transporter_toslink:
...
+- **Total songs**: Number of music files currently in service.
```
Comment (c0ffeeca7) [inline source/_integrations/squeezebox.markdown]:
```suggestion
- **Total songs**: Total number of music files currently in service.
```

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]
Tiny tweaks

File: source/_integrations/squeezebox.markdown (hunk 2 above)
Comment (pssc) [reply source/_integrations/squeezebox.markdown]: So its seconds but duration auto scales, to display SS:MM:HH on larger collections

Comment (c0ffeeca7) [reply source/_integrations/squeezebox.markdown]: thank you for clarifying. Just to be sure about the order: it's  SS:MM:HH, and not HH:MM:SS ?

Comment (pssc) [reply source/_integrations/squeezebox.markdown]: you are correct I transposed it in my comment 1433:35:25 on my prod system.

File: source/_integrations/squeezebox.markdown [hunk 8]
```diff
@@ -56,6 +57,17 @@ transporter_toslink:
...
 - **Library rescan**:  The music library is currently being scanned by LMS (depending on the type of scan, some content may be unavailable).
 
+### Sensors
+
+- **Last scan**: Date of the last library scan.
+- **Player count**: Number of players on the service.
+- **Player count off service**: Number of players not on this service.
+- **Total albums**: Total number of albums currently available in the service.
+- **Total artists**: Total number of artists currently available in the service.
+- **Total duration**: Duration of all tracks in service.
```
Comment (c0ffeeca7) [inline source/_integrations/squeezebox.markdown]:
```suggestion
- **Total duration**: Duration of all tracks in service (HHHH:MM:SS).
```
ah, ok. So it's like this?

Comment (pssc) [reply source/_integrations/squeezebox.markdown]: HHHH:MM:SS as I understand it

Comment (c0ffeeca7) [reply source/_integrations/squeezebox.markdown]: oh, I see. Thank you.

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @pssc :+1:

**Pull request merged**